### DISCUSSION TITLE
SiLU for motif discovery?

### DISCUSSION THREAD

**Discussion by colobas** [jzhoulab/puffin#9] [Q&A] [answered]
First of all, congrats on a great paper.

I'm curious about your claim that using a SiLU activation in the first stage of training facilitates motif discovery. Is this something you found empirically? Could you comment on this or point me to references that suggest or support this?

Thanks in advance and congrats again!

**Comment by jzthree** [comment, marked as answer]
This motivation of SiLU-like activation functions was to avoid zero-gradients e.g. https://arxiv.org/pdf/1710.05941v1.pdf  and we empirically found it to help motif discovery in the first stage.

**Comment by colobas** [comment]
Thank you!
